src/dmqnode/request/ConsumeRequest.d

```diff
@@ -212,7 +212,10 @@ public scope class ConsumeRequest : Protocol.Consume, StorageEngine.IConsumer
                 this.finish_trigger = true;
                 break;
 
-            default:
+            case Deletion, None:
+                assert(false);
```

Comment:
So if there are some legal enum values that are expected to never ever happen, why `final switch` at all? Simply to ensure new values will be added?

Reply:
In a later commit this is changed to `verify(false)`.
This is a matter of taste. I use `final switch` where possible to show that all cases are considered.

Reply:
> In a later commit this is changed to verify(false)

Aha, makes sense in that case :+1: